content/applications/essentials/contacts.rst

```diff
@@ -178,13 +178,64 @@ section, and other partner options, including :guilabel:`Partner Activation` and
 Review` configurations. These are **only** present when the *Enterprise Resellers* module is
 installed.
 
+To publish a list of all active partners on the website, a new webpage must be created first.
+Navigate to :menuselection:`Website app --> Site --> Pages` and click :guilabel:`New`. Hover over
+the :guilabel:`Blank Page` template, and click the :guilabel:`Use this template` button to create a
+new page.
+
+In the :guilabel:`New Page` pop-up window, enter `Partners` in the :guilabel:`Page Title` field.
+Then, click :guilabel:`Create`. In the :guilabel:`Edit` toolbar on the far-right of the page, click
+:guilabel:`Save`.
+
+Return to :menuselection:`CRM app --> Sales --> Customers` and click the Kanban card for a partner.
```

Comment:
```suggestion
Next, return to :menuselection:`CRM app --> Sales --> Customers`, and click the Kanban card for a partner.
```
OPTIONAL: adding 'next' to keep the reader in the "flow" of the process being detailed//added a comma for enhanced sentence flow/pacing.